README.md

```diff
@@ -16,20 +16,4 @@ For more specific topics, try a SIG.
 ---
 Note that this repository is loosely modeled on the kubernetes open governance repository at https://github.com/kubernetes/community.  The goal is to apply the similar governance within the Cray / HPE community.  Phase 1 will seed the governance and Phase 2 will expand participation.
 
-There will be two phases for the rollout of over-all governance for the Cray / HPE community:
-
-### Phase 1 
-Our initial model will be very lean and simple. Community participants may submit PR's against any one of the published repositories. Once the PR is submitted, the Cray internal engineering management for that component/repo will review the PR and determine its suitability. Accepted PR's will be merged to master and become part of a subsequent release within the context of release cadence as it exists as of November 2020. Decisions regarding community contributions to a given repository will be at the sole discretion of the Cray engineering organization during this phase.
```

Comment:
Cray->HPE

Reply:
This appears to be a deletion of old stuff, being replaced by a link to the original prose at its source.  So nothing really to fix here I think though maybe the original source could use the fix!